src/components/NavBar.tsx

```diff
@@ -249,7 +251,7 @@ export default (props: Props) => {
                     <Typography variant="subtitle2">Pricing</Typography>
                   </Link>
                   <HorizontalSpacer size={24} />
-                  <a href="https://docs.kintohub.com">
+                  <a target="_blank" href="https://docs.kintohub.com">
```

Comment:
action: there should be some warning about `rel="noopener noreferrer"` no?

Reply:
didnt get any